coffee_time_firmware/lib/coffee_time_firmware/boiler/temp_control.ex

```diff
@@ -149,4 +163,46 @@ defmodule CoffeeTimeFirmware.Boiler.TempControl do
     Logger.info("Changing duty cycle: #{cycle}")
     :ok = Boiler.DutyCycle.set(context, cycle)
   end
+
+  def adjust_hold_mode(%{hold_mode: :backoff} = data, _) do
+    data
+  end
+
+  def adjust_hold_mode(data, value) do
+    if data.target_temperature - value > 10 do
```

Comment:
`10` here should be based off the offset instead of hardcoded. Also any comments would be good.